packages/analytics/src/events.ts

```diff
@@ -1,3 +1,4 @@
 export enum Events {
   NewSubscriber = 'New Subscriber',
+  ConfirmedSubscriber = 'Confirmed Subscriber',
```

Comment:
Are you using this event somewhere?

I think this is still wrong.

New subscriber -> User added its email
Confirmed Subscriber -> User accessed the URL that was sent by the confirmation email.

@JoaoVictor6 is this currently happening?